README.md

```diff
@@ -380,7 +380,46 @@ Staltz can see Arj has invited Mix because he's replicating Arj's
 
 ## 5. Security Considerations
 
-TBD
+### 5.1. Degraded confidentiality at scale
+
+As noted in the [private-groups-original-notes], SSB Private Groups are based on
+symmetric encryption and static keys (no key rotation), which means it cannot
+guarantee [perfect-forward-secrecy] neither [post-compromise-security]. This
+makes SSB Private Groups less likely to guarantee confidentiality at scale, when
+a group has dozens or hundreds of members. If any of the members leaks the group
+secret, all past communication in the group is compromised.
```

Comment:
Another thing that I remember keks mentioned is that our ever increasing logs is also a potential security liability. That said, my believe is that metafeeds allows us to colletively forget something if we choose to. And this ties into membership removal as well.

Reply:
I think that once we work on member removal, we should update this spec repo to add member removal flows to create new epochs, etc. And then at that point we can also update the Security Considerations, because technically an eavesdropper won't know the difference between a new group epoch and a new group, and this will make analysis attacks harder. And then at that point we can also mention that forgetting older epochs is a way of slowly/gradually achieving forward secrecy (although not guaranteeing it).

Reply:
Yep, agree